app/src/main/java/com/example/playlistmakervmp/SettingsActivity.kt

```diff
@@ -17,6 +19,16 @@ class SettingsActivity : AppCompatActivity() {
         backButton.setOnClickListener { finish() }
 
 
+        val themeSwitcher = findViewById<com.google.android.material.switchmaterial.SwitchMaterial>(R.id.themeSwitcher)
```

Comment:
У тебя есть в 11 строке импорт нужного класса `SwitchMaterial`, поэтому тут можно не указывать полный путь до класса:
```suggestion
        val themeSwitcher = findViewById<SwitchMaterial>(R.id.themeSwitcher)
```